daemon/cmd/endpoint.go

```diff
@@ -709,6 +709,14 @@ func (d *Daemon) DeleteEndpoint(id string) (int, error) {
 	} else if err = endpoint.APICanModify(ep); err != nil {
 		return 0, api.Error(DeleteEndpointIDInvalidCode, err)
 	} else {
+		containerID := ep.GetShortContainerID()
+		if containerID != "" {
+			log.WithField("containerID", containerID).Info("Delete endpoint request")
```

Comment:
maybe containerID ->  logfields.ContainerID for consistency.

Reply:
OK. Thanks!